<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FloatingPoint</class>
 <widget class="QDialog" name="FloatingPoint">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>804</width>
    <height>265</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Floating Point</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>70</y>
     <width>49</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Input: </string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>430</x>
     <y>70</y>
     <width>49</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Input: </string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>150</y>
     <width>49</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Output: </string>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>420</x>
     <y>150</y>
     <width>49</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Output: </string>
   </property>
  </widget>
  <widget class="QLineEdit" name="txtSIN">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>70</y>
     <width>16</width>
     <height>24</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">#txtSIN{
padding-left: 2px;
background-color: white;
color: black;

}</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="maxLength">
    <number>1</number>
   </property>
  </widget>
  <widget class="QLineEdit" name="txtExpIN">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>70</y>
     <width>61</width>
     <height>24</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">#txtExpIN{
padding-left: 2px;
background-color: white;
color: black;

}</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="maxLength">
    <number>8</number>
   </property>
  </widget>
  <widget class="QLineEdit" name="txtFIN">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>70</y>
     <width>151</width>
     <height>24</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">#txtFIN{
padding-left: 2px;
background-color: white;
color: black;

}</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="maxLength">
    <number>23</number>
   </property>
  </widget>
  <widget class="QLineEdit" name="txtDecOut">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>150</y>
     <width>241</width>
     <height>24</height>
    </rect>
   </property>
   <property name="readOnly">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>200</y>
     <width>80</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>Run</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="txtSOUT">
   <property name="geometry">
    <rect>
     <x>470</x>
     <y>150</y>
     <width>16</width>
     <height>24</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">#txtSOUT{
padding-left: 2px;
background-color: white;
color: black;

}</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="maxLength">
    <number>1</number>
   </property>
   <property name="readOnly">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLineEdit" name="txtExpOUT">
   <property name="geometry">
    <rect>
     <x>490</x>
     <y>150</y>
     <width>61</width>
     <height>24</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">#txtExpOUT{
padding-left: 2px;
background-color: white;
color: black;

}</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="maxLength">
    <number>8</number>
   </property>
   <property name="readOnly">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLineEdit" name="txtFOUT">
   <property name="geometry">
    <rect>
     <x>560</x>
     <y>150</y>
     <width>151</width>
     <height>24</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">#txtFOUT{
padding-left: 2px;
background-color: white;
color: black;

}</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="maxLength">
    <number>23</number>
   </property>
   <property name="readOnly">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLineEdit" name="txtDecIN">
   <property name="geometry">
    <rect>
     <x>470</x>
     <y>70</y>
     <width>241</width>
     <height>24</height>
    </rect>
   </property>
   <property name="maxLength">
    <number>18</number>
   </property>
   <property name="readOnly">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_2">
   <property name="geometry">
    <rect>
     <x>480</x>
     <y>200</y>
     <width>80</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>Run</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_3">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>200</y>
     <width>80</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>Clear</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_4">
   <property name="geometry">
    <rect>
     <x>620</x>
     <y>200</y>
     <width>80</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>Clear</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
